typings/index.d.ts

```diff
@@ -1333,6 +1333,12 @@ declare module 'discord.js' {
 		public unban(user: UserResolvable, reason?: string): Promise<User>;
 	}
 
+	export class GuildBanStore extends DataStore<Snowflake, BanInfo> {
+		constructor(guild: Guild);
+		public add(ban: BanInfo, cache: boolean): BanInfo;
```

Comment:
cache is optional, or at least should be, to stay consistent with other add methods